base/docs/helpdb/Base.jl

```diff
@@ -1821,6 +2018,19 @@ length(collection)
 
 Returns the index of the last value in `a` less than or equal to `x`, according to the
 specified order. Returns `0` if `x` is less than all values in `a`.
+
+# Examples
+
+```jldoctest
+julia> searchsortedlast([1, 2, 4, 5, 4], 4)
```

Comment:
examples for this function should use sorted inputs, that's what it's designed for